Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get install -y vim curl
 
 WORKDIR /src/conjur-api
 
-COPY Gemfile conjur-api.gemspec ./
+COPY Gemfile conjur-api.gemspec VERSION ./
```

Comment:
I've added a comment `Reference point.` at the point where the VERSION file is dynamically generated based on the latest CHANGELOG entry.  The above change was needed because of the introduction of the VERSION file that's dynamically generated (was previously hardcoded in version.rb)

Reply:
Ah, thanks for the explanation!